ext/DifferentiationInterfaceReverseDiffExt.jl

```diff
@@ -34,6 +30,15 @@ function DI.value_and_pullback!(
     return y, dx
 end
 
+function DI.value_and_pullback!(
+    _dx, backend::AutoReverseDiff, f, x::X, dy::Y, extras::Nothing=nothing
+) where {X<:Number,Y}
+    x_array = [x]
+    dx_array = [zero(x)]
```

Comment:
or `dx_array = zero(x_array)`.

Reply:
doesn't really matter since `x` is scalar

Reply:
but we could use a `FillArray`

Reply:
Wouldn't that clash with `Duplicate`?

Reply:
I don't think so but let's keep normal arrays for now